app/components/node/node.html

```diff
@@ -0,0 +1,262 @@
+<rd-header>
+  <rd-header-title title="Node details">
+    <a data-toggle="tooltip" title="Refresh" ui-sref="node({id: node.Id})" ui-sref-opts="{reload: true}">
+      <i class="fa fa-refresh" aria-hidden="true"></i>
+    </a>
+  </rd-header-title>
+  <rd-header-content>
+    <a ui-sref="swarm">Swarm Nodes</a> > <a ui-sref="node({id: node.Id})">{{ node.Id }}</a>
```

Comment:
I think we should display the node hostname here instead of the node ID (easier to read in the navigation breadcrumb).